test/shared/utils.py

```diff
@@ -36,12 +37,12 @@ def cibuildwheel_run(project_path, env=None, add_env=None):
     if add_env is not None:
         env.update(add_env)
 
-    subprocess.check_call(
+    p = subprocess.check_call(
```

Comment:
What are these changes?

Reply:
I will remove this. Artefact after debug.

Reply:
Oh, don't worry. Sorry to be too quick about it!